scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -669,6 +662,21 @@ public void jobStateUpdatedEvent(NotificationData<JobInfo> notification) {
         }
     }
 
+    @Override
+    public void jobUpdatedFullDataEvent(JobState job) {
+        Iterator<SchedulerEventListenerExtended> it = eventListeners.iterator();
+        while (it.hasNext()) {
+            SchedulerEventListenerExtended l = it.next();
+            try {
+                l.jobUpdatedFullDataEvent(job);
+            } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)